SCClassLibrary/JITLib/GUI/TaskProxyGui.sc

```diff
@@ -240,8 +240,12 @@ TaskProxyGui : JITGui {
 	}
 
 	openDoc { |strings, bounds|
-		var doc = strings.join.newTextWindow("edit me");
-		try { doc.bounds_(bounds ? Rect(0, 400, 400, 200)) };
+        var doc = Document.new("edit" + super.getName, strings.join, false);
+		try {
+            doc.front;
+            doc.promptToSave = false;
+            doc.bounds_(bounds ? Rect(0, 400, 400, 200))
```

Comment:
While you are at it, could you please write it in a more optimized way:
```
doc.bounds_(bounds ?? { Rect(0, 400, 400, 200) })
```

Reply:
Otherwise, great, thank you!

Reply:
Good call, thank you!